terraform/application/application.tf

```diff
@@ -17,9 +17,12 @@ module "application_configuration" {
       PGSSLMODE          = local.postgres_ssl_mode
       CANONICAL_HOSTNAME = local.canonical_hostname
   })
-  secret_variables = {
-    DATABASE_URL = module.postgres.url
-  }
+  secret_variables = merge(
+    {
+      DATABASE_URL        = module.postgres.url
+      HEARTBEAT_CHECK_URL = module.statuscake[0].heartbeat_check_urls[local.heartbeat_check_name]
```

Comment:
This may break if var.enable_monitoring is false. We should only add `{HEARTBEAT_CHECK_URL = module.statuscake[0].heartbeat_check_urls[local.heartbeat_check_name]}` if it's true